tools/build_rules/prelude_bazel

```diff
@@ -0,0 +1,11 @@
+load(
+    "@io_bazel_rules_pex//pex:pex_rules.bzl",
+    "pex_binary",
+    "pex_library",
+    "pex_test",
+    "pex_pytest",
+)
+load(
+    "@bazel_tools//tools/build_defs/pkg:pkg.bzl",
```

Comment:
with 0.3.0 ```@bazel_tools``` is not supported I believe. Our repo can compile both in bazel 0.2.3 and bazel 0.3.0. Can you please ensure that it compiles with bazel 0.3.0 as well?